app/views/layouts/_header.html.erb

```diff
@@ -17,8 +17,8 @@
     <!-- Collect the nav links, forms, and other content for toggling -->
     <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
       <ul class="nav navbar-nav">
-        <li class="active"><a href="/">Home<span class="sr-only">(current)</span></a></li>
-        <li><a href="#">Popular</a></li>
+        <li><%= active_link_to 'Home', root_path, wrap_tag: 'li', active: :exclusive %></li>
+        <li><%= active_link_to 'Popular', popular_products_path, wrap_tag: 'li', active: :exclusive %></li>
```

Comment:
@cchanningallen https://github.com/comfy/active_link_to/blob/master/lib/active_link_to/active_link_to.rb#L38 I think this answers your question, but maybe I'm not understanding it correctly?